### DISCUSSION TITLE
French translation for TimVer_0.9.0.0

### DISCUSSION THREAD

**Discussion by logounet** [Timthreetwelve/TimVer#38] [General]
Hello,

I have updated your French translation for TimVer_0.9.0.0 (corrected errors and titled...) if you accept it. Attached is the file Strings.fr-FR.xaml.zip

[Strings.fr-FR.xaml.zip](https://github.com/user-attachments/files/15760333/Strings.fr-FR.xaml.zip)

Best regards

**Comment by Timthreetwelve** [comment]
@logounet Thank you so much for updating the French language file.  It will be in the next release.

I'm creating an issue from this discussion so that I can track it.

Thanks again,
Tim
